### ISSUE TITLE
yaml parsing in file ... not working for me

### ISSUE THREAD

**Issue by maybegeek** [PhileCMS/Phile#279]
Hi there,

I'm using the downloaded version 1.6, everything works except the yaml header parsing.

file looks something like this:

```
---
Title: value1
Description: value2
Template: value3
Nochwas: value4
Dinge: value5
Apfel: value6
Birne: value7
---

here comes the content. here comes the content. 
```

in `config.php` I have:

```
$config['plugins']['phile\\parserMeta'] = ['active' => true];
$config['plugins']['phile\\parserMeta'] = ['format' => 'YAML'];
$config['plugins']['phile\\phpFastCache'] = ['active' => 'false'];
$config['plugins']['phile\\simpleFileDataPersistence'] = ['active' => 'false'];
```

but also tested with:

```
$config['plugins']['phile\\parserMeta']['active'] = true;
$config['plugins']['phile\\parserMeta']['format'] = 'YAML';
$config['plugins']['phile\\phpFastCache']['active'] = false;
$config['plugins']['phile\\simpleFileDataPersistence']['active'] = false;
```

the default theme does not pick up on `meta.description` nor `meta.title`, instead showing the yaml meta info in `{{ content }}` ... as string. Plus strange behavior of last written yaml meta pair, which gets its own `<h2>` tag. 

perhaps a pointer to the right documentation for yaml in production would help,

thanks alot!

**Comment by JazzMaster** [comment]
leave format at 'Phile'. It works fine.

**Comment by Schlaefer** [comment]
Up until now including the 1.6 release the metadata parser only detects `<!-- … -->` and `/* … */` fences for meta tags. The development branch added supported for custom/configurable fences (and with that `--- … ---`) in #277.

So you have to download the unreleased development version if you want to use `--- … ---` right now.

**Comment by Frodox** [comment]
And this feature will be released when #284 will be merged.

**Comment by Schlaefer** [comment]
Is going to work in 1.7.0
